webui/react/src/pages/Templates/TemplateCreateModal.tsx

```diff
@@ -137,7 +173,7 @@ const TemplateCreateModalComponent: React.FC<Props> = ({ workspaceId, onSuccess
             },
           ]}>
           <CodeEditor
-            file={''}
+            file={template?.config ? yaml.dump(template.config) : ''}
```

Comment:
**question:** does `yaml.dump` do any sort of validation or processing? can it throw an error? If not we are good, but otherwise it would need a `try/catch` block to capture it or some error boundary (not sure if there is one yet)

Reply:
This should be alright because the proto we use can make sure `template.config` is a valid json object